src/templates/rule.js

```diff
@@ -3,111 +3,32 @@ import Layout from '../components/layout/'
 import { graphql } from 'gatsby'
 import showdown from 'showdown'
 import {
+	getChangelog,
```

Comment:
Note to self:
- link to diff of the file eg(#commithash..../_rules/filename.md)